osgtest/library/voms.py

```diff
@@ -148,6 +148,7 @@ def is_installed():
         if not core.dependency_is_installed(dep):
             return False
 
+    # TODO: drop this check when 3.3 is completely EOL
```

Comment:
We can drop this entire library and all corresponding tests once 3.3 is EOL

Reply:
We won't be testing VOMS at all? Not just voms-admin?

Reply:
Yea, not really. Maybe `voms-proxy-fake` but we don't need the library for it or really any of what happens in `test_50_voms.py`